### DISCUSSION TITLE
For support, please use mailing lists or Slack

### DISCUSSION THREAD

**Discussion by lhawthorn** [instructlab/instructlab#1042] [Announcements]
We're planning to use GitHub discussions for documenting things like nasty bugs with work in progress rather than Q&A type requests. Please choose the best mailing list for your inquiry or visit us on Slack.

You can find all the ways to reach project maintainers beyond GitHub on our [Collaboration page](https://github.com/instructlab/community/blob/main/Collaboration.md).

**Comment by sankapoo** [comment]
I am trying ilab utility to train granite-7b model on my mac laptop using ilab model train command. It takes 6-7 hours to complete the full training. But when I chat with that trained model I am not getting any response that I supposed to get from trained model as per my training data. I am getting gguf files getting generated at ~/.local/share/instructlab/checkpoints/hf_format/samples_56/pytorch_model.gguf

I have tried following command to train the model: -

ilab model train --data-path ~/.local/share/instructlab/datasets/skills_train_msgs_2025-01-16T03_45_04.jsonl --model-path ~/Desktop/instructlab/mymodel/granite-7b-lab  --ckpt-output-dir ~/.local/share/instructlab/checkpoints/ --iters 10

ilab model train --data-path ~/.local/share/instructlab/datasets/knowledge_train_msgs_2025-01-16T03_45_04.jsonl --model-path instructlab/granite-7b-lab  --ckpt-output-dir ~/.local/share/instructlab/checkpoints/

But finally I am not getting any trained response from trained model (~/.local/share/instructlab/checkpoints/hf_format/samples_56/pytorch_model.gguf) 

I am using ilab, version 0.22.2. 

Can you please help me where I am making mistake. Or there is some bug in instructlab. 

With Regards
Sandeep

**Comment by ValerieKum** [comment]
What is your slack channel please?
